vipps-getting-started.md

```diff
@@ -57,6 +60,27 @@ Document version: 3.3.4.
 
 <!-- END_TOC -->
 
+## Product types
```

Comment:
What do you think about these descriptions?
I put them in Getting Started to help people get oriented, but they could also go on the terminology page.